lib/octokit/client/milestones.rb

```diff
@@ -28,7 +28,7 @@ def list_milestones(repository, options = {})
       # @param repository [String, Repository, Hash] A GitHub repository.
       # @param options [Hash] A customizable set of options.
       # @option options [Integer] :milestone Milestone number.
-      # @option options [String] :state (open) State: <tt>open</tt> or <tt>closed</tt>.
+      # @option options [String] :state (open) State: <tt>open</tt>, <tt>closed</tt>, or <tt>all</tt>.
```

Comment:
Oops, everything but `number` is existing copypasta. Mind removing `:state`, `:sort`, and `:direction here?